oauth_provider/runtests/urls.py

```diff
@@ -13,9 +13,9 @@ def resource_None_scope_view(request):
     return HttpResponse()
 
 
-urlpatterns = patterns('',
```

Comment:
According to the 1.7 docs:
> A function that takes a prefix, and an arbitrary number of URL patterns, and returns a list of URL patterns in the format Django needs.
So it always returned a list, but could also add a prefix (the first argument that was `''` in this case). So I think this does exactly the same as the `patterns` function did. I'm confident to merge it :)